pkg/controller/operators/catalog/step_ensurer.go

```diff
@@ -111,7 +111,7 @@ func (o *StepEnsurer) EnsureSecret(operatorNamespace, planNamespace, name string
 
 // EnsureBundleSecret creates user-specified secrets from the bundle. Called when StepResource.Secret is true
 func (o *StepEnsurer) EnsureBundleSecret(namespace string, secret *corev1.Secret) (status v1alpha1.StepStatus, err error) {
-	_, createErr := o.kubeClient.KubernetesInterface().CoreV1().Secrets(namespace).Create(secret)
+	_, createErr := o.kubeClient.KubernetesInterface().CoreV1().Secrets(namespace).Create(context.TODO(), secret, metav1.CreateOptions{})
```

Comment:
Is this change a result of the bump to kube 1.18 in the next commit?

Reply:
Yep!